include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -432,24 +462,32 @@ template <class T>
 void gemmComputeW2(MatrixT<T>& w2, ConstPanelT<Coord::Col, T>& w, ConstPanelT<Coord::Col, T>& x) {
   using matrix::unwrapExtendTiles;
   using tile::internal::gemm_o;
-
-  const auto ex = getHpExecutor<Backend::MC>();
+  using pika::execution::experimental::keep_future;
+  using pika::threads::thread_priority;
 
   // Note:
   // Not all ranks in the column always hold at least a tile in the panel Ai, but all ranks in
   // the column are going to participate to the reduce. For them, it is important to set the
   // partial result W2 to zero.
-  pika::dataflow(ex, unwrapExtendTiles(tile::internal::set0_o), w2(LocalTileIndex(0, 0)));
+  tile::set0(dlaf::internal::Policy<Backend::MC>(thread_priority::high),
```

Comment:
To do: create a variable `policy_hp = Policy<Backend::MC>(thread_priority::high)`.